types/expect-puppeteer/index.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for expect-puppeteer 2.2
+// Project: https://github.com/smooth-code/jest-puppeteer
```

Comment:
Use this link is better: https://github.com/smooth-code/jest-puppeteer/tree/master/packages/expect-puppeteer

Reply:
Updated in the other PR too